polaris-react/src/components/Tooltip/components/TooltipOverlay/TooltipOverlay.tsx

```diff
@@ -7,37 +7,59 @@ import {
   PositionedOverlay,
 } from '../../../PositionedOverlay';
 import {useI18n} from '../../../../utilities/i18n';
+import {getRectForNode} from '../../../../utilities/geometry';
+import type {TooltipPosition} from '../../Tooltip';
 
 import styles from './TooltipOverlay.scss';
 
 export interface TooltipOverlayProps {
   id: string;
   active: boolean;
-  preventInteraction?: PositionedOverlayProps['preventInteraction'];
   preferredPosition?: PositionedOverlayProps['preferredPosition'];
   children?: React.ReactNode;
   activator: HTMLElement;
   accessibilityLabel?: string;
   onClose(): void;
+  transform: TooltipPosition;
```

Comment:
We may want to rename this prop here. Within the `PositionedOverlay` we have a `transform` prop which is a `string`, whereas here it is an object of values. This could be confusing for folks. Maybe something like `coordinates` would work better here given the type of values we're passing.